### PR TITLE
Add Gentoo wiki search engine

### PR DESCRIPTION
**Pull request by aliceinwire**
Add Gentoo wiki search engine

### CODE REVIEW

**Comment by kvch** [comment]
Thanks for your contribution! Could you please address the issues detected by PEP8?

**Comment by aliceinwire** [comment]
@kvch @asciimoo fixed pep8 issues

File: searx/engines/gentoo.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        'base': 'https://wiki.gentoo.org',
+        'search': '/index.php?title=Special:Search&offset={offset}&{query}\
+                &profile=translation&languagefilter={language}'
+    }
+}
+
+
+# get base & search URLs for selected language
+def get_lang_urls(language):
+    print(language)
```
Comment (asciimoo) [inline searx/engines/gentoo.py]: Please remove this debug print.

**Review by asciimoo — CHANGES_REQUESTED** [review]
Thanks for the PR. Please fix that one minor issue and it's ready to merge.

File: searx/engines/gentoo.py (hunk 1 above)
Comment (aliceinwire) [reply searx/engines/gentoo.py]: fixed :)

**Review by asciimoo — APPROVED** [review]
Cool, thanks!

**Pull request merged**